vars/envs.yml

```diff
@@ -1,7 +1,7 @@
 ---
 environment_vars:
   TEST_NAME: ayyylmao
-  DATABASE_HOST: localhost
+  DATABASE_HOST: 127.0.0.1
   DATABASE_NAME: "{{ postgres_db_name }}"
   DATABASE_USERNAME: "{{ postgres_db_user }}"
   DATABASE_PASSWORD: "{{ postgres_db_password }}"
```

Comment:
Technically a new Rails app expects this ENV VAR to be APP_NAME_DATABASE_PASSWORD but since we have to load all the other ENV VARs into database.yml anyway there's probably no need to change it